README_CHECKS.md

```diff
@@ -13,6 +13,7 @@
 | networkpolicy-targets-pod | NetworkPolicy | Makes sure that all NetworkPolicies targets at least one Pod | default |
 | pod-probes | Pod | Makes sure that all Pods have bot a readinessProbe and a libenessProbe configured | default |
 | container-security-context | Pod | Makes sure that all pods have good securityContexts configured | default |
+| pod-seccomp-profile | Pod | Makes sure that all pods have a Seccomp profile configured | optional |
```

Comment:
This should be `container-seccomp-profile`, to match the name of the check as registered in the code (`"Container Seccomp Profile"`).

Reply:
Definitely, this should be consistent. Already updated.